SMLHelper/Handlers/CraftDataHandler_BelowZero.cs

```diff
@@ -195,93 +195,64 @@ void ICraftDataHandler.AddLinkedItems(TechType techType, ICollection<TechType> l
         /// Make sure your mod is loading after the mod whose TechData you are trying to access.
         /// </summary>
         /// <param name="techType">The TechType whose TechData you want to access.</param>
-        /// <returns>The ITechData from the modded item if it exists; Otherwise, returns <c>null</c>.</returns>
+        /// <returns>The RecipeData from the item if it exists; Otherwise, returns <c>null</c>.</returns>
```

Comment:
Opening up a comment thread so it can be resolved later.

I'm basically in agreement with @zorgesho .
If this change is to de-duplicate code, we should take the opportunity to do as much as possible.
It will make things cleaner and easier to maintain in the long run.
Not to mention that it'll save us from copy/paste errors as well.

If you need any help with refactoring, let me know and I can jump on.

Reply:
Refactors made.